plugins/modules/openssh_cert.py

```diff
@@ -111,15 +111,17 @@
             - "The point in time the certificate is valid from. Time can be specified either as relative time or as absolute timestamp.
                Time will always be interpreted as UTC. Valid formats are: C([+-]timespec | YYYY-MM-DD | YYYY-MM-DDTHH:MM:SS | YYYY-MM-DD HH:MM:SS | always)
                where timespec can be an integer + C([w | d | h | m | s]) (for example C(+32w1d2h)).
-               Note that if using relative time this module is NOT idempotent."
+               Note that if using relative time this module is NOT idempotent. To ignore this value during comparison with an existing certificate set 
+               I(ignore_timestamps=true)."
             - Required if I(state) is C(present).
         type: str
     valid_to:
         description:
             - "The point in time the certificate is valid to. Time can be specified either as relative time or as absolute timestamp.
                Time will always be interpreted as UTC. Valid formats are: C([+-]timespec | YYYY-MM-DD | YYYY-MM-DDTHH:MM:SS | YYYY-MM-DD HH:MM:SS | forever)
                where timespec can be an integer + C([w | d | h | m | s]) (for example C(+32w1d2h)).
-               Note that if using relative time this module is NOT idempotent."
+               Note that if using relative time this module is NOT idempotent. To ignore this value during comparison with an existing certificate set 
+               I(ignore_timestamps=true)."
```

Comment:
I would split this up into a new paragraph:
```suggestion
               Note that if using relative time this module is NOT idempotent."
            - To ignore this value during comparison with an existing certificate set I(ignore_timestamps=true).
```